### ISSUE TITLE
Download pretrained model

### ISSUE THREAD

**Issue by GeorgeXia1828** [apache/incubator-mxnet#3218]
Sorry to ask the problem here, but I don't know how to get the pretrained model vgg 16 after read the caffe documents, because I don't know  what is the gist id crresponding to model. Can anyone help?

> [rd@idc01-rank-gpu-01 scripts]$ ./download_model_from_gist.sh 
> usage: download_model_from_gist.sh <gist_id> <dirname>

**Comment by tornadomeet** [comment]
@George19890617 you can download the vgg16 of mxnet in the readme of https://github.com/dmlc/mxnet/tree/master/example/fcn-xs 
the http is https://pan.baidu.com/s/1bgz4PC#path=%252Ffcn-xs, name is  VGG_FC_ILSVRC_16_layers-***

**Comment by George19890617** [comment]
@tornadomeet thanks so much, I found that seems to be the networking problem.

**Comment by George19890617** [comment]
@tornadomeet I found it write in 
https://github.com/dmlc/mxnet/tree/master/example/fcn-xs 

> (when using the newest mxnet, you'd better using larger learning rate, such as 1e-4, 1e-5, 1e-6 instead, because the newest mxnet will do gradient normalization in SoftmaxOutput)

But isn't it the default is set to not do the normalization of gradients? I am not clear about the quote sentence means.

**Comment by tornadomeet** [comment]
yes, the mxnet version right now will do it auto, but the model i provide at that time will not.

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
